import java.util.Scanner;

public class ArithmeticExceptionDemo
{

	public static void main(String[] args) 
	{
		ArithmeticExceptionDemo  ae = new ArithmeticExceptionDemo (); 
		ae.divide();
		System.out.println("Check Me");

	}
	private void divide() 
	{
	    Scanner sc = new Scanner(System.in);
	    System.out.println("Enter two numbers ");
	    
	    int no1 = sc.nextInt();
	    int no2 = sc.nextInt();
	    
	    try 
	    {
	    	 System.out.println(no1/no2);
	    }
	    catch(ArithmeticException ae)
	    {
		     System.out.println("Check no2. It should not be zero.");
		     no2 = sc.nextInt();
		     System.out.println(no1/no2);
	    }
	}

}

